### PR TITLE
Add new mappings for numlock, numpadenter and numpadcomma on X11

### PR DESCRIPTION
**Pull request by blaind**
- [x] Tested on all platforms changed
- [x] Compilation warnings were addressed
- [x] `cargo fmt` has been run on this branch
- [x] `cargo doc` builds successfully
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [ ] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [ ] Created or updated an example program if it would help users understand this functionality
- [ ] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

### CODE REVIEW

**Comment by ArturKovacs** [comment]
We are hoping to release the new keyboard API in the next version of winit, which should include all of these. So right now, I would instead encourage you @blaind, to join us in testing #1932, if you can.

**Comment by maroider** [comment]
> We are hoping to release the new keyboard API in the next version of winit, which should include all of these.

I have half a mind to close this PR, #1266, #1846, and #1864 on this basis.

**Comment by ogoffart** [comment]
I think this could merged as-is. It is kind of trivial, and the small bug it fixes does not need to wait that the new keyboard API is finished.

**Comment by ArturKovacs** [comment]
Yeah we can merge this. @maroider do you have anything against this?

**Comment by ogoffart** [comment]
ping?

**Comment by maroider** [comment]
Welp, I completely forgot about this.

I don't have anything in particular against this, but I suspect you'll have to take some extra steps to ensure the changelog entry ends up in the right place.

**Comment by ArturKovacs** [comment]
@maroider good point. I suggest rebasing this on master.

**Comment by blaind** [comment]
> @maroider good point. I suggest rebasing this on master.

Was about to rebase, but made a mistake and now too many commits are visible in the pull request. Will fix later today.

**Comment by maroider** [comment]
> Was about to rebase, but made a mistake and now too many commits are visible in the pull request. Will fix later today.

Indeed. It looks like you did a merge instead of a rebase.

**Comment by blaind** [comment]
> It looks like you did a merge instead of a rebase.

Should be fixed now. Based on the master branch & force-pushed

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- On X11, add mappings for numpad comma, numpad enter, numlock and pause
```
Comment (maroider) [inline CHANGELOG.md:3]:
```suggestion
- On X11, add mappings for numpad comma, numpad enter, numlock and pause.
```
Teeny tiny nitpick: Most of our changelog entries end with a period.

**Review by ArturKovacs — APPROVED** [review]

**Pull request merged**